mllib/src/main/scala/org/apache/spark/mllib/optimization/Gradient.scala

```diff
@@ -61,14 +62,27 @@ abstract class Gradient extends Serializable {
 class LogisticGradient extends Gradient {
   override def compute(data: Vector, label: Double, weights: Vector): (Vector, Double) = {
     val margin = -1.0 * dot(data, weights)
-    val gradientMultiplier = (1.0 / (1.0 + math.exp(margin))) - label
+    /**
+     * gradientMultiplier = (1.0 / (1.0 + math.exp(margin))) - label
```

Comment:
Is it the logistic function? Maybe we should make create a function for it as well.